src/components/v5/shared/Card/Card.tsx

```diff
@@ -6,15 +6,26 @@ import { CardProps } from './types';
 const displayName = 'v5.Card';
 
 const Card = React.forwardRef<HTMLDivElement, CardProps>(
-  ({ hasShadow, rounded = 's', children, className, ...props }, ref) => (
+  (
+    {
+      hasShadow,
+      rounded = 's',
+      children,
+      className,
+      isTopSectionWithBackground,
+      ...props
+    },
+    ref,
+  ) => (
     <div
       className={clsx(
         className,
-        'bg-base-white border border-gray-200 p-6 flex flex-col w-full',
+        'bg-base-white border border-gray-200 flex flex-col w-full',
         {
           'rounded-lg': rounded === 's',
           'rounded-xl': rounded === 'm',
           'shadow-default': hasShadow,
+          'p-6': !isTopSectionWithBackground,
```

Comment:
As I understand, this prop isTopSectionWithBackground adds only paddings to the component. I think it could be better to name it more like: 'withPaddings'